gait-control/PneumaticBoard.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun 29 10:44:14 2017
+
+@author: jrs
+#python 3
+"""
+
+import serial # python -m pip install pyserial
+from time import sleep
+
+class PneumaticBoard:
+    def __init__(self, actuators=2, port='COM3', baud=9600):
+        self.pwmList = [0] * actuators
+        self.ser = serial.Serial(port, baud)
+        print ("Please wait for control board connection...")
+        sleep(5)
+        print ("Done.")
+		
+    def on(self):
+        print ("Valves on.")
+        self.ser.write(b'y')
+	
+    def off(self):
+        print ("Valves off.")
+        self.ser.write(b'n')
+		
+    def close(self):
+        self.ser.close()
+		
+    def setPWMList(self, pwmList):
+        self.pwmList = pwmList
+        self.updatePWM()
+
+    def setPWM(self, pwm, index):
+        self.pwmList[index] = pwm
+        self.updatePWM()
+		
+    def updatePWM(self):
+        command = b'v '
```

Comment:
Only concern here is that we lose generality for actuators that are not 2-input with bidirectional bending. Don't worry about it for now, but this file will only be compatible with snake/bidirectional actuators.